JS-Code-Challenge/coding-String-challenge.js

```diff
@@ -22,11 +22,7 @@ function findNumberOfChar(str) {
     for (let i = 0; i < str1.length; i++) {
         let char = str1[i];
         if (char !== ' ') {
-            if (charCount[char]) {
-                charCount[char]++;
-            } else {
```

Comment:
Please remove existing code only if it is wrong. 
You can add alternative of that as shortcut